### ISSUE TITLE
Add syscall package

### ISSUE THREAD

**Issue by FlorianUekermann** [golang/appengine#67]
The syscall package contains a large number of useful constants. Please consider adding a syscall package that contains those constants.

**Comment by asac** [comment]
This caused troubles with aws-go-sdk which I use for making S3 presigned urls on go appengine. Please see https://github.com/aws/aws-sdk-go/issues/1190 for that discussion...

Thanks for looking into this!

**Comment by SebScoFr** [comment]
Any update on this?
Currently the [viper library](https://github.com/spf13/viper) is unusable:
```
ERROR    2018-11-28 15:55:17,981 instance_factory.py:232] Failed to build Go application: (Executed command: /private/etc/google-cloud-sdk/platform/google_appengine/goroot-1.9/bin/go-app-builder -app_base /Users/seb/Sites/go/src/alpha/api/app -api_version go1 -arch 6 -dynamic -goroot /private/etc/google-cloud-sdk/platform/google_appengine/goroot-1.9 -nobuild_files ^^$ -incremental_rebuild -unsafe -print_extras_hash app.go)

2018/11/28 15:55:17 go-app-builder: Failed parsing input: parser: bad import "syscall" in github.com/spf13/afero/afero.go from GOPATH
```

**Comment by broady** [comment]
Available in the Go 1.11 beta runtime.
